src/app/pages/dashboard/components/widget-backup/widget-backup.component.ts

```diff
@@ -58,6 +58,9 @@ export class WidgetBackupComponent extends WidgetComponent implements OnInit {
   backups: BackupRow[] = [];
   isLoading = false;
 
+  successStates = [JobState.Success, JobState.Finished];
+  failedStates = [JobState.Failed, JobState.Error, JobState.Aborted];
```

Comment:
Instead of having two arrays, where we can accidentally add the same state twice, we can just create one array and check the absence of state in that array to determine other states.
```
  get failedCount(): number {
    return this.backups.filter((backup) => !this.successStates.includes(backup.state)).length;
  }

  private getTile(title: string, tasks: BackupRow[]): BackupTile {
    const successfulTasks = tasks.filter((backup) => this.successStates.includes(backup.state));
  ...
```

Reply:
I created two arrays because the absence of "success" in the array does not necessarily mean that the task is "unsuccessful," and vice versa. There are `JobState` statuses that are not related to either success or failure. 

<img width="388" alt="image" src="https://github.com/truenas/webui/assets/20611516/582121a0-c691-4e48-a073-863f0569e38e">

Am I missing something?